### PR TITLE
16.0 mig project forecast line

### PR DESCRIPTION
**Pull request by otmanelagy**
migration of module project_forecast_line to version 16

### CODE REVIEW

**Comment by otmanelagy** [comment]
i have always problems when passing tests on sale.order and hr.leave related to invisible and readonly fields on version 16 , fields not exist in views but it existing on xml file and on local while testing, for that i commented those test . thanks for checking @yankinmax

**Comment by emagdalenaC2i** [comment]
Please squash your commits

**Comment by yankinmax** [comment]
@otmanelagy if you want to pass the tests you need to look for the changes in the views and correct them during migration.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

File: project_forecast_line/tests/test_forecast_line.py [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+                "sequence": 1,
+            }
+        )
+
+        cls.product_dev_tm = cls.env["product.product"].create(
+            {
+                "name": "development time and material",
+                "detailed_type": "service",
+                "service_tracking": "task_in_project",
+                # "price": 95,
```
Comment (leemannd) [inline project_forecast_line/tests/test_forecast_line.py:62]: Is there a reason to comment this?

**Comment by leemannd** [comment]
Hello @otmanelagy are you still working on it?
There have been a few improvements in version 15.0. Is it possible to have them into the migration aswell?

**Pull request closed without merging**